examples/evaluation/huggingface-evaluation.ipynb

```diff
@@ -0,0 +1,1340 @@
+      "score: Your numerical score for the model's answer_quality based on the rubric\n",
+      "justification: Your step-by-step reasoning about the model's answer_quality score\n",
+      "    )\n"
+     ]
+    }
+   ],
+   "source": [
+    "from mlflow.metrics.genai import EvaluationExample, make_genai_metric\n",
+    "\n",
+    "answer_quality_definition = \"\"\"Please evaluate answer quality for the provided output on the following criteria: fluency, clarity, and conciseness. Each of the criteria is defined as follows:\n",
+    "  - Fluency measures how naturally and smooth the output reads.\n",
+    "  - Clarity measures how understandable the output is.\n",
+    "  - Conciseness measures the brevity and efficiency of the output without compromising meaning.\n",
+    "The more fluent, clear, and concise a text, the higher the score it deserves.\n",
+    "\"\"\"\n",
+    "\n",
+    "answer_quality_rubric = \"\"\"Answer quality: Below are the details for different scores:\n",
+    "  - Score 1: The output is entirely incomprehensible and cannot be read.\n",
+    "  - Score 2: The output conveys some meaning, but needs lots of improvement in to improve fluency, clarity, and conciseness.\n",
+    "  - Score 3: The output is understandable but still needs improvement.\n",
+    "  - Score 4: The output performs well on two of fluency, clarity, and conciseness, but could be improved on one of these criteria.\n",
+    "  - Score 5: The output reads smoothly, is easy to understand, and clear. There is no clear way to improve the output on these criteria.\"\"\"\n",
+    "\n",
+    "example1 = EvaluationExample(\n",
+    "    input=\"What is MLflow?\",\n",
+    "    output=\"MLflow is an open-source platform. For managing machine learning workflows, it including experiment tracking model packaging versioning and deployment as well as a platform simplifying for on the ML lifecycle.\",\n",
+    "    score=2,\n",
+    "    justification=\"The output is difficult to understand and demonstrates extremely low clarity. However, it still conveys some meaning so this output deserves a score of 2.\",\n",
+    ")\n",
+    "\n",
+    "example2 = EvaluationExample(\n",
+    "    input=\"What is MLflow?\",\n",
+    "    output=\"MLflow is an open-source platform for managing machine learning workflows, including experiment tracking, model packaging, versioning, and deployment.\",\n",
+    "    score=5,\n",
+    "    justification=\"The output is easily understandable, clear, and concise. It deserves a score of 5.\",\n",
+    ")\n",
+    "\n",
+    "answer_quality_metric = make_genai_metric(\n",
+    "    name=\"answer_quality\",\n",
+    "    definition=answer_quality_definition,\n",
+    "    grading_prompt=answer_quality_rubric,\n",
+    "    version=\"v1\",\n",
+    "    examples=[example1, example2],\n",
+    "    model=\"openai:/gpt-4\",\n",
+    "    greater_is_better=True,\n",
+    ")\n",
+    "\n",
+    "print(answer_quality_metric)"
```

Comment:
I would prefer not printing the metric here, as the output is super long. We can briefly explain what this metric does here, and can print the metric after finishing the evaluation.